guides/get-started/transactions.md

```diff
@@ -55,8 +55,10 @@ var transaction;
 // the transaction fee when the transaction fails.
 server.loadAccount(destinationId)
   // If the account is not found, surface a nicer error message for logging.
-  .catch(StellarSdk.NotFoundError, function (error) {
-    throw new Error('The destination account does not exist!');
+  .catch(function (error) {
+    if (error instanceof StellarSdk.NotFoundError) {
+      throw new Error('The destination account does not exist!');
+    }
```

Comment:
Needs to return the error if not an `instanceof StellarSdk.NotFoundError` otherwise the error will get caught here and not follow through to any subsequent catch blocks

Reply:
10114e7090499f23d0d3273c54b0d548f3066d01

Reply:
Good call